clearwater-snmpd/etc/snmp/snmpd.conf.clearwater-snmpd

```diff
@@ -259,3 +259,7 @@ dlmod chronos_handler /usr/lib/clearwater/chronos_handler.so
 # Don't log on successful connections - allowing this log means that a log is
 # made to syslog on every stats query, which is too spammy
 dontLogTCPWrappersConnects 1
+
+# File descriptor MIB handling
+pass .1.2.826.0.1.1578918.17.1 /bin/bash
```

Comment:
Doesn't this need to pass to something more than just '/bin/bash'? Not clear how this works, but feels odd. 
Also, blank line at end of file